tagr/storage/aws.py

```diff
@@ -48,7 +49,7 @@ def dump_json(self, df_metadata, proj, experiment, tag):
             ContentType="application/json",
         )
 
-    def dump_pickle(self, pickle_object, proj, experiment, tag, filename):
+    def dump_pickle(self, model, proj, experiment, tag, filename):
```

Comment:
this is a little confusing since we pickle more than just models. iirc anything that's not a df gets pickled